### DISCUSSION TITLE
Post Judge QA

### DISCUSSION THREAD

**Discussion by JeeberC4** [code-423n4/2022-11-redactedcartel-findings#419] [Announcements]
## 🧑‍⚖️ Post-judging QA; deadline: 01/09/2023 09:00AM PST

1. Please post feedback about *specific findings* on the relevant issue in this repo.
2. More general feedback for the judge can be shared here. 

The judge for this contest is @Picodes   

## Reminders

- Please keep your comments as fact-based as possible.
- C4 judges have final authority in determining validity and severity.
- You are welcome to start new discussions here!
- Please link to specific issues wherever relevant. 

Thank you!

**Comment by 0xSmartContract** [comment]
First of all I would like to thank @Picodes for a great work on this judge 

It seems that the following QA finding can be added to Medium finding scope 174;


https://github.com/code-423n4/2022-11-redactedcartel-findings/issues/174



https://github.com/code-423n4/2022-11-redactedcartel-findings/blob/main/data/0xSmartContract-Q.md#l-10-hardcode-the-address-causes-no-future-updates

**Reply by Picodes** [reply]
Thanks, @0xSmartContract. However, I don't think it qualifies for an upgrade: here the medium issue is that the address is incorrect, not the lack of upgrade method.

**Comment by rvierdiiev** [comment]
Hello @Picodes.
I would like to ask why [198](https://github.com/code-423n4/2022-11-redactedcartel-findings/issues/198) is not considered as high in this case? As this attack will fully block migration.

**Reply by Picodes** [reply]
Hi @rvierdiyev, to me blocking the migration is a medium issue according to https://docs.code4rena.com/awarding/judging-criteria: "Assets not at direct risk, but the function of the protocol or its availability could be impacted, or leak value with a hypothetical attack path with stated assumptions, but external requirements."

Assets are not directly at risk, and the functionality of the protocol (the possibility to migrate) is impacted.

**Reply by rvierdiiev** [reply]
Thanks for the explanation.
